src/sentry/integrations/discord/client.py

```diff
@@ -118,13 +122,22 @@ def track_response_data(
         extra: Mapping[str, str] | None = None,
     ) -> None:
         """
-        For all Discord api responses this:
-        - Sends response metrics to Datadog
-        - Sends response info to logs
+        Handle response from Discord by logging and capturing metrics
         """
-        discord_error_code = "no_error_code"
-        code_message = ""
-        include_in_slo = True
+        log_params = {
+            "code": code,
+            "error": str(error),
+            "extra": extra,
+        }
+
+        if self.integration_type:
+            log_params[str(self.integration_type)] = self.name
+
+        try:
+            logging_context = getattr(self, "logging_context", None)
+            log_params["logging_context"] = logging_context
```

Comment:
when is `logging_context` populated? what does it mean?

Reply:
This is carry over from the existing functionality, I think it comes from up the chain